src/NuGet.Clients/NuGet.Console/Xamls/ConsoleContainer.xaml.cs

```diff
@@ -75,6 +80,26 @@ public void NotifyInitializationCompleted()
 
         public void Dispose()
         {
+            if (!_isTelemetryEmitted)
+            {
+                var telemetryEvent = new TelemetryEvent("PackageManagerConsoleLoadCount", new Dictionary<string, object>
```

Comment:
I think it's better if you think about this in a `1 event, lots of data` way. 

See https://github.com/NuGet/Client.Engineering/issues/319. 

That way analysis will be easier because you only need to query for 1 event.

Reply:
Ok. I combined 2 events in `ConsoleContainer.xaml.cs` into 1 event.